src/util.js

```diff
@@ -0,0 +1,66 @@
+import window from 'global/window';
+import {EMBED_TYPE_IFRAME} from './constants';
+
+const {encodeURIComponent} = window;
+
+let BASE_URL = 'https://players.brightcove.net/';
+
+/**
+ * Gets the URL to a player on CDN.
+ *
+ * @private
+ * @param  {Object} params
+ *         A parameters object. See README for details.
+ *
+ * @return {string}
+ *         A URL.
+ */
+const getUrl = (params) => {
+  const iframe = params.embedType === EMBED_TYPE_IFRAME;
+  const ext = iframe ? 'html' : 'min.js';
```

Comment:
Do we want to have an option to get an unminified player?

Reply:
Hmm, maybe as a feature in a future release.